### PR TITLE
Azure release 11.0.0 with Kubernetes 1.16.3

### PR DESCRIPTION
**Pull request by tfussell**
Split from https://github.com/giantswarm/releases/pull/37

### CODE REVIEW

File: azure.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    version: 1.0.0
+  - name: azure-operator
+    version: 2.8.0
+  - name: cert-operator
+    version: 0.1.0
+  - name: chart-operator
+    version: 0.7.0
+  - name: cluster-operator
+    provider: azure
+    version: 0.21.1
```
Comment (MarcelMue) [inline azure.yaml]: This is still uncertain - we might need `0.21.2` to include changes by @mazzy89

**Comment by MarcelMue** [comment]
Please review the `changelog` from your perspective - both wording and content.

**Comment by tfussell** [comment]
Do you think we should highlight the deny-all network policy higher up in the notes? I guess customers won't really be running workloads in giantswarm or kube-system namespaces, but I don't know for sure.

**Comment by MarcelMue** [comment]
> Do you think we should highlight the deny-all network policy higher up in the notes? I guess customers won't really be running workloads in giantswarm or kube-system namespaces, but I don't know for sure.

I wasn't sure about this - maybe it makes sense to just highlight it in the message going to slack? Uncertain.

**Comment by tfussell** [comment]
Maybe we can discuss how customers use these namespaces (if at all) tomorrow.

**Comment by MarcelMue** [comment]
I added sig-product to the reviewers to get some feedback on these notes. Please note any improvements here.

**Comment by teemow** [comment]
@Oshratn the important thing here is of course the Kubernetes Upgrade. This is our first 1.16 release. It's late. Additionally we can maybe highlight the new default deny all network policy to further lock down clusters by default.

**Comment by Oshratn** [comment]
Those were exactly my thoughts. I was wondering why 1.16 was stuck way down on the list.
Shall I craft the opening paragraph I suggested?

**Comment by MarcelMue** [comment]
I have added a little intro text at the top which was provided by @tfussell .

The message to be sent out through slack would be:
```
:zap: Giant Swarm Release 11.0.0 for Azure is now active for you! :zap:

This is the first Giant Swarm release including Kubernetes 1.16. As part of this release, we have also removed CPU limits from several supporting components and adjust priority classes to ensure system reliability under heavy load. More details about changes to individual components and potentially breaking changes can be found on GitHub [LINK HERE].

*Please read the release notes carefully!*
```

File: release-notes/azure/v11.0.0.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+## :zap: Giant Swarm Release 11.0.0 for Azure is now active for you! :zap:
+
+This is the first Giant Swarm release including Kubernetes 1.16. As part of this release, we have also removed CPU limits from several supporting components and adjust priority classes to ensure system reliability under heavy load. More details about changes to individual components can be found below.
```
Comment (tfussell) [inline release-notes/azure/v11.0.0.md]:
```suggestion
This is the first Giant Swarm release including Kubernetes 1.16. As part of this release, we have also removed CPU limits from several supporting components and adjusted priority classes to ensure system reliability under heavy load. More details about changes to individual components can be found below.
```

File: release-notes/azure/v11.0.0.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+**Note**: This release includes a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks traffic to and from pods in the `giantswarm` and `kube-system` namespaces to improve security. GS components have modified to work under this environment, but any other workload pods in these namespaces may cease to function without a network policy allowing ingress and egress.
```
Comment (tfussell) [inline release-notes/azure/v11.0.0.md]:
```suggestion
**Note**: This release includes a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks traffic to and from pods in the `giantswarm` and `kube-system` namespaces to improve security. GS components have been modified to work under this environment, but any other workload pods in these namespaces may cease to function without a network policy allowing ingress and egress.
```

File: release-notes/azure/v11.0.0.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+## :zap: Giant Swarm Release 11.0.0 for Azure is now active for you! :zap:
+
+This is the first Giant Swarm release which includes Kubernetes v1.16. In addition, CPU limits have been removed from several supporting components and priority classes have been adjusted to ensure system reliability under heavy load. Further details about changes to individual components can be found below.
+
+### Important upgrade notes
+- This release includes a new [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks network traffic to and from pods in the `giantswarm` and `kube-system` namespaces by default to improve security. GS components have been modified to work under this environment, but any other pods in the cluster communicating with these system pods may cease to function without a network policy allowing traffic.
```
Comment (puja108) [inline release-notes/azure/v11.0.0.md]:
```suggestion
- This release includes a new [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks network traffic to and from pods in the `giantswarm` and `kube-system` namespaces by default to improve security. GS components have been modified to work under this environment, but any other pods in the cluster communicating with these system pods may cease to function without a network policy allowing traffic. Thus, if you are running pods that need to talk to system pods, you will need to allow those explicitly by adding a [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/) for them.
```

File: release-notes/azure/v11.0.0.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- [As previously communicated](https://github.com/giantswarm/giantswarm/blob/master/news/2019/10/product/k8s_1.16_breaking_changes.md), Kubernetes v1.16 has removed several API groups which were already deprecated. Please ensure that all affected components have been updated to the 1.16 compatible versions before upgrading clusters from v1.15.
+- Previous Giant Swarm releases included Tiller v2.14 in the `giantswarm` namespace which is not compatible with Kubernetes v1.16. This Tiller used by Giant Swarm to manage the cluster has therefore been updated to the Kubernetes v1.16-compatible v2.16. Any Tiller instances deployed in other namespaces will likewise need to be updated to v2.16 or greater to continue to function after an upgrade.
```
Comment (puja108) [inline release-notes/azure/v11.0.0.md]:
```suggestion
- Previous Giant Swarm releases included Tiller v2.14 in the `giantswarm` namespace which is not compatible with Kubernetes v1.16. This Tiller - used by Giant Swarm to manage the cluster - has therefore been updated to the Kubernetes v1.16-compatible v2.16. Any Tiller instances deployed in other namespaces will likewise need to be updated to v2.16 or greater to continue to function after an upgrade.
```

File: release-notes/azure/v11.0.0.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+This is the first Giant Swarm release which includes Kubernetes v1.16. In addition, CPU limits have been removed from several supporting components and priority classes have been adjusted to ensure system reliability under heavy load. Further details about changes to individual components can be found below.
+
+### Important upgrade notes
+- This release includes a new [network policy](https://docs.giantswarm.io/guides/limiting-pod-communication-with-network-policies/#default-policies) which blocks network traffic to and from pods in the `giantswarm` and `kube-system` namespaces by default to improve security. GS components have been modified to work under this environment, but any other pods in the cluster communicating with these system pods may cease to function without a network policy allowing traffic.
+- [As previously communicated](https://github.com/giantswarm/giantswarm/blob/master/news/2019/10/product/k8s_1.16_breaking_changes.md), Kubernetes v1.16 has removed several API groups which were already deprecated. Please ensure that all affected components have been updated to the 1.16 compatible versions before upgrading clusters from v1.15.
```
Comment (puja108) [inline release-notes/azure/v11.0.0.md]:
```suggestion
- [As previously communicated](https://github.com/giantswarm/giantswarm/blob/master/news/2019/10/product/k8s_1.16_breaking_changes.md), Kubernetes v1.16 has removed several API groups which were already deprecated. Please ensure that you have migrated your deployments and pipelines from the deprecated to the new API groups. You can check the above-linked breaking change communication and consult with your Solution Engineer for help with that.
```

**Comment by cokiengchiara** [comment]
1. Thinking about business stakeholders (cc @foschubert @Oshratn ), would it be helpful to include one line describing what each operator is? To put the changes and improvement for each operator into context. Or maybe one paragraph at the top describing the business value is enough.

(Just an idea, if Ludacris is in a hurry now, we can implement for next time)

2. WDYT about creating separate section for GS-internal users only to the bottom? So it's easier for a customer to read only what is more relevant to him / her.

**Comment by MartaD** [comment]
@cokiengchiara I like these suggestions but vote for doing it next time.

**Comment by MarcelMue** [comment]
>     1. Thinking about business stakeholders (cc @foschubert @Oshratn ), would it be helpful to include one line describing what each operator is? To put the changes and improvement for each operator into context. Or maybe one paragraph at the top describing the business value is enough.
> 
>     2. WDYT about creating separate section for GS-internal users only to the bottom? So it's easier for a customer to read only what is more relevant to him / her.
> 
> 
> These are just ideas. If Ludacris is in a hurry now, I don't want to delay you. We can implement next time. Maybe we can have a separate retro focused just on how to improve release notes (the content only).

1. Maybe could be helpful - some docs in general could be helpful that are then just linked in the release notes. AFAIK no other project / service explains each component in each release.

2. All of this is customer facing - this information is already here because we think it is valuable for the customer (at different levels of technical proficiency). Changes with no customer impact / strictly GS internal have already been omitted IMO.

**Comment by MarcelMue** [comment]
I would like to merge this today - please approve if there are no suggestions / uncertainties.

**Comment by Oshratn** [comment]
Since done is better than perfect. I think the intro added does the trick for now.

**Pull request merged**